packages/sdk/src/wasm/credentialDeploymentTransactions.ts

```diff
@@ -302,5 +302,5 @@ export async function signCredentialTransaction(
     signingKey: HexString
 ): Promise<HexString> {
     const digest = getCredentialDeploymentSignDigest(credDeployment);
-    return Buffer.from(await ed.signAsync(digest, signingKey)).toString('hex');
+    return Buffer.from(ed.sign(digest, signingKey)).toString('hex');
```

Comment:
**Minor thing:** Why use sync here? It seems to make very little sense when the function is still async

Reply:
To avoid breaking changes. Async functions use webcrypto under the hood, which is not supported on react native. My first attempt included polyfilling webcrypto, but that was not straightforward.